src/components/ComponentIndex.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const ComponentIndex = ({ id, name, reactLink, angularLink, htmlLink }) => (
+  <div className="component-index" id={id}>
+    <h2 className="sprk-b-TypeDisplayThree component-index__heading">{name}</h2>
+    <ul className="component-index-list">
+      { reactLink
+      && (
+      <li className="component-index-list__item">
```

Comment:
Could change this to use one <li> and update the parts of the URL that need to be updated.